encoder/vaapiencoder_h264.cpp

```diff
@@ -495,6 +620,19 @@ class VaapiEncStreamHeaderH264
         bsToHeader(m_sps, bs);
     }
 
+    /*
+        void setSubSetSps(const VAEncSequenceParameterBufferH264* const
+       sequence, VideoProfile profile)
+        {
+            ASSERT(m_sps.empty());
+            BitWriter bs;
+            bit_writer_write_sps (&bs, sequence, profile);
+            bit_writer_write_svc_extension (&bs, sequence, profile);
+            bit_writer_write_vui_extension (&bs, sequence, profile);
+            bit_writer_write_addtional_extension (&bs, sequence, profile);
+            bsToHeader(m_sps, bs);
+        }
+    */
```

Comment:
Seem it is not necessary for CQP, I am not sure it is necessary for CBR case, so keep it here right now.